src/components/Error.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components/macro";
+
+function Message() {
```

Comment:
I would name it `ErrorMessage` or `Error` as well 🙂

Reply:
Changed it. Thx!